src/main/resources/schema/ans/0.10.0/story_elements/custom_embed.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.0/story_elements/custom_embed.json",
+  "title": "Custom Embed",
+  "description": "A custom embed element. Can be used to reference content from external providers about which little is known.",
+  "type": "object",
+  "additionalProperties": false,
+
+  "properties": {
+    "type": {
+      "type": "string",
+      "enum": [ "custom_embed" ]
+    },
+    "_id": {
+      "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.0/traits/trait_id.json"
+    },
+    "subtype": {
+      "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.0/traits/trait_subtype.json"
+    },
+    "channels": {
+      "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.0/traits/trait_channel.json"
+    },
+    "alignment": {
+      "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.0/traits/trait_alignment.json"
+    },
+    "additional_properties": {
+      "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.0/traits/trait_additional_properties.json"
+    },
+
+    "embed": {
+      "title": "Embed",
+      "description": "The embed data.",
+      "type": "object",
+      "additionalProperties": false,
+
+      "properties": {
+        "id": {
+          "title": "Embed ID",
+          "description": "Foreign ID of embedded item.",
+          "type": "string",
+          "maxLength": 128,
+          "minLength": 1
+        },
+
+        "url": {
+          "title": "Embed Provider URL",
+          "description": "Provider URL for this embed item. When concatenated with Embed ID, should produce a URL that returns json metadata about the embedded content.",
+          "type": "string",
+          "maxLength": 128,
+          "minLength": 1
+        },
+
+        "config": {
+          "title": "Embed Configuration",
+          "description": "Arbitrary configuration data generated by a plugin. Users are responsible for maintaining schema.",
+          "type": "object",
+
+          "additionalProperties": false,
+          "patternProperties": {
+            "^(?!(referent|type)$)[a-zA-Z0-9_]*$": {}
```

Comment:
what about `version` field or it's okay to have it?

Reply:
@yury-sannikov Good catch; just updated it